src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>Does the note about the 2016 change qualify anything from our analysis? If not, I would remove.</p><p> </p><p>I would also make sure the bullet points line up before and after "based on the information above". For instance, your pointing out that there are outliers in<code>trip_distance</code> and then saying these values should be excluded is great.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>